layout/common/scripts.jsx

```diff
@@ -32,12 +33,15 @@ module.exports = class extends Component {
         return <Fragment>
             <script src={cdn('jquery', '3.3.1', 'dist/jquery.min.js')}></script>
             <script src={cdn('moment', '2.22.2', 'min/moment-with-locales.min.js')}></script>
+            {/* <script src={cdn('pjax', '0.2.8', 'min/pjax.min.js')}></script> */}
+            <script src={pjaxMinJsPath}></script>
             {clipboard && <script src={cdn('clipboard', '2.0.4', 'dist/clipboard.min.js')} defer></script>}
             <script dangerouslySetInnerHTML={{ __html: `moment.locale("${language}");` }}></script>
             <script dangerouslySetInnerHTML={{ __html: embeddedConfig }}></script>
-            <script src={url_for('/js/column.js')}></script>
+            <script data-pjax src={url_for('/js/column.js')}></script>
             <Plugins site={site} config={config} page={page} helper={helper} head={false} />
-            <script src={url_for('/js/main.js')} defer></script>
+            <script data-pjax src={url_for('/js/main.js')} defer></script>
+            <script src={url_for('/js/pjax.js')}></script>
```

Comment:
We should add an option to let user enable/disable pjax.

Reply:
Let me fix it.